common/org.testeditor.dsl.common.ui/src/org/testeditor/dsl/common/ui/utils/ProjectContentGenerator.xtend

```diff
@@ -142,6 +146,25 @@ class ProjectContentGenerator {
 		]
 	}
 
+	def createGradleSettings(IProject project, IProgressMonitor monitor) {
+		var Properties props = System.getProperties();
+		if (!isNullOrEmpty(props.getProperty("http.proxyHost"))) {
+			var IFile gradleProperties = project.getFile("gradle.properties")
```

Comment:
In another project we worked with gradle.properties for user settings. We checked only a template file in. the gradle.properties are ignored. 
I think this belongs to a project setup managed by a developer. In the current setup managed by the test editor we doesn't support some team share actions.
For the moment without team share support, I think the solution helps.